x-pack/plugins/security_solution/public/common/mock/global_state.ts

```diff
@@ -500,4 +501,26 @@ export const mockGlobalState: State = {
    */
   management: mockManagementState as ManagementState,
   discover: getMockDiscoverInTimelineState(),
+  notes: {
+    ids: ['1'],
+    entities: {
+      '1': {
+        eventId: 'event-id',
+        noteId: '1',
+        note: 'note-1',
+        timelineId: 'timeline-1',
+        created: 1663882629000,
+        createdBy: 'elastic',
+        updated: 1663882629000,
+        updatedBy: 'elastic',
+        version: 'version',
+      },
+    },
+    status: {
+      fetchNotesByDocumentId: ReqStatus.Idle,
```

Comment:
This might not apply currently but something to keep in mind: If we ever want to show two lists of notes at the same time, this status pattern would need to be nested one layer deeper and contain an identifier, e.g. the documentId.

```js
status: {
  fetchNotesByDocumentId: {
    "2398hf93hf2ncsdf0": ReqStatus.Idle,
  }
},
```

Reply:
yes that could definitely happen, but I felt like I wanted to keep this PR as simple as possible and working for the needs we need today. We can easily update this when the need arise (if ever)